LICENSE

```diff
@@ -1,4 +1,4 @@
-This software is copyright (c) 2024 by Timothy Legge <timlegge@gmail.com>.
+This software is copyright (c) 2025 by Timothy Legge <timlegge@gmail.com>.
```

Comment:
I use year ranges, e.g. "2024-2025" but that's a preference, I guess.

Reply:
I usually do too - since it is less than 12 months old I had not added to - range thanks

Reply:
Weird thats generated - I wonder by what

Reply:
figured it out - if Dist:Zilla does not find it it inserts the current year

Reply:
sorting in the release